libraries/chain/block_header_state.cpp

```diff
@@ -183,7 +183,7 @@ void evaluate_finalizer_policies_for_promotion(const block_header_state& prev,
 void finish_next(const block_header_state& prev,
                  block_header_state& next_header_state,
                  vector<digest_type> new_protocol_feature_activations,
-                 instant_finality_extension if_ext,
+                 finality_extension if_ext,
```

Comment:
Maybe we should stop using `if_ext`, using `fin_ext` or `f_int` instead.

Reply:
Personally, I'd prefer avoiding such naming changes, I don't find `if_ext` objectionable, but I guess opinions may vary.